### ISSUE TITLE
Create dedicated fields for expected/target milestone

### ISSUE THREAD

**Issue by jpmedley** [GoogleChrome/chromium-dashboard#1473]
**Is your feature request related to a problem? Please describe.**
Feature owners sometimes list this information in text fields. 

* Updating such information cannot be automated. A global search and replace would capture text like that shown below. It would also rewrite text such as, "in Chrome 94 we did such-and-such." It could also affect references such as, "there are 95 occurrences."
* Implementing automated verification requests presents a choice between missing references and being spammy. Not reaching text fields for version numbers would mean no update request would be sent for legitimate references like the one shown below. Searching for such numbers would mean sending emails for false positives like those listed above. 

![image](https://user-images.githubusercontent.com/8204171/129758458-97d69715-fb30-42b3-8928-e13377f64a60.png)

**Describe the solution you'd like**
Dedicated expected milestone fields for dev trial, origin trial, shipping, deprecation, and removal.

**Additional context**

- A clear indication of expected milestones opens the door to distinguishing between plans and promises in the [milestone visualization](https://github.com/GoogleChrome/chromium-dashboard/pull/1469).
- Knowing that a feature _might_ ship in a month gives writers time to properly prepare related content. 
- With an expected milestone, feature owners could approve for a specific version, letting the system automigrate the milestone from expected to actua.

**Comment by jpmedley** [comment]
Here is a concrete use case for this feature. We've started experimenting with engineers contributing drafts for later use on MDN. Before a feature ships, drafts are held in [an appropriate GitHub repo](https://github.com/WICG/idle-detection/commit/231c86037a55ff41aca13512d485f8718fa2d809). As shipping approaches, I have to [ask the feature owner](https://groups.google.com/a/google.com/g/chrome-status-discussion-archive/c/D6Ecv-8MI9M/m/csWiocexAQAJ) to alert me when they think the feature is ready to ship. 

To make the transition, the feature owner has to remember to alert me, and I need to track all changes that might shipping and remember to do the conversion.

**Comment by jpmedley** [comment]
**Use Case:** The releases page could make a distinction between things that definitely are shipping and things that only might be shipping.

**Comment by jyasskin** [comment]
**User Story:** I was trying to ask Microsoft their opinion on https://chromestatus.com/feature/5694001745231872, but at the time it showed "Shipped: 93", and I didn't want to confuse things by sending them confidently-incorrect information. If it had shown "estimated to have shipped: 93" or even just no shipping date once the estimated branch point passes with no update, I could have used the chromestatus entry as the canonical ID for the feature.

Basically, data in chromestatus is going to get out of date, and the fact that it's out of date needs to be as obvious as possible to humans reading the pages.

**Comment by cwilso** [comment]
We should have a separate "Target" or expected milestone field (from the actual shipping field) - and enforce that it is filled out before an intent to ship is generated.

**Comment by jpmedley** [comment]
I found another instances where not having this is a problem. This feature had an expected start of an origin trial in 89; however, the [intent to experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/vkS3k30lWNs/m/Gt4sKECQEgAJ) (still called intent to prototype when it went out) was never approved. 

![image](https://user-images.githubusercontent.com/8204171/148995297-106a035e-211d-4e74-a4b6-dc0e663dbeca.png)

**Comment by jpmedley** [comment]
Different design approach is being worked on, so closing.
